Formula/rsync.rb

```diff
@@ -15,35 +14,64 @@ class Rsync < Formula
     sha256 "cc355e9492df6bab4952e835dd8575de86fa169f6451218c5d2c777c4d9462a5" => :high_sierra
   end
 
-  depends_on "autoconf" => :build
+  depends_on "lz4"
+  depends_on "openssl@1.1"
+  depends_on "popt"
+  depends_on "xxhash"
+  depends_on "zstd"
+
+  uses_from_macos "zlib"
 
   # hfs-compression.diff is marked by upstream as broken as of 3.1.3
```

Comment:
maybe update the version of this line to 3.2.0?

Reply:
How about something like `hfs-compression.diff has been marked by upstream as broken since 3.1.3 and has not been reported fixed as of 3.2.0`? So we include both the initial report and last check.

Reply:
yeah, that would be much better :) 👍